src/main/java/com/marklogic/kafka/connect/source/AbstractSourceTask.java

```diff
@@ -0,0 +1,66 @@
+package com.marklogic.kafka.connect.source;
+
+import org.apache.kafka.connect.source.SourceRecord;
+import org.apache.kafka.connect.source.SourceTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Base class for concrete SourceTask implementations, providing some generic functionality.
+ */
+abstract class AbstractSourceTask extends SourceTask {
```

Comment:
Let's not make this until we have a second type of source task (odds are we won't, at least not for awhile).